### DISCUSSION TITLE
how can i join the paddleOCR wechat group and slack channel? and download the trained model

### DISCUSSION THREAD

**Discussion by audo23** [PaddlePaddle/PaddleOCR#8296] [General]
Dear all:
how can i join the paddleOCR wechat group and slack channel? and download the trained model ? the QR code and the slack channel provided  are all expired,,,
I wish to download the pretrained model about form recognization / invoice recognization to see the performance and hope to make a contribution.

**Comment by sheepsheepyoung** [comment]
WeChat is a popular communication application in China but international developers may not be familiar with it, so we open a new slack channel. Two platforms are both available.https://join.slack.com/t/paddleocr/shared_invite/zt-1jjk36xdj-id4A7gMRMRjbeVlQKZI3_g the slack invitation will be expired on December 14th

**Reply by TuanBC** [reply]
Could you update another invitation link to the slack group? I would love to join the group but cannot find anyway now.

**Comment by sainivedh19pt** [comment]
Hi @sheepsheepyoung,
Could you please update the slack invitation link for international community to join in.

Thanks

**Reply by TasneemVKhan** [reply]
Hi, You can try this [link]( https://join.slack.com/t/paddleocr/shared_invite/zt-1n7yulykm-yiM3vjrKR7ZPOtrrdHx7Twl)

**Reply by TuanBC** [reply]
@TasneemVKhan Unfortunately the link here is also invalid too.

**Reply by TasneemVKhan** [reply]
Send me your email ID. I'll send you an invitation over it.

**Reply by TasneemVKhan** [reply]
https://join.slack.com/t/paddleocr/shared_invite/zt-1nm72m16p-VH2oWaIodKqh5LUmWlqcCw 

See if this works

**Reply by TuanBC** [reply]
@TasneemVKhan Thank you a lot I have joined the group successfully now ^^

**Reply by TasneemVKhan** [reply]
No issues. This link is open for 30 days anyone can join it.

**Reply by soham24** [reply]
@TasneemVKhan hello 
Can you send a new invitation link please

**Comment by gradypalfrey** [comment]
Is there another new invitation link to the slack channel?

**Comment by TuanBC** [comment]
Actually, the Slack channel right now is a bit inactive, but if you want to join, here is the slack invitation link:

https://join.slack.com/t/paddleocr/shared_invite/zt-1xeay78i8-pQfJ2EC5PijiD3uCKOSbow

**Comment by soham24** [comment]
Thank you sir.

Actually I was looking for downloading this material:
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/applications/README_en.md#2

But as you said the slack channel is inactive these days, I can’t find it
there either. Can you help me with it please?
On Tue, 13 Jun 2023 at 6:49 AM, Tuan Nguyen ***@***.***>
wrote:

> Actually, the Slack channel right now is a bit inactive, but if you want
> to join, here is the slack invitation link:
>
>
> https://join.slack.com/t/paddleocr/shared_invite/zt-1xeay78i8-pQfJ2EC5PijiD3uCKOSbow
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PaddlePaddle/PaddleOCR/discussions/8296#discussioncomment-6158057>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAOND735MEVSG46TDWKXDW3XK65YXANCNFSM6AAAAAAR7RL4HE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
Regards
Soham Railkar <http://facebook.com/soham24>

**Comment by timothee-dunglas** [comment]
Hello all, I have the same issue as soham24 stated. I want to download this material : https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/applications/README_en.md#2

Specifically I am interested in VAT invoice, ID card recognition and Universal card recognition. Can someone help me to download these models ? I cannot find them in the wechat.

Regards
Timothee

**Reply by Marbyun** [reply]
hi bro are you already get the model?

**Comment by enggabrielhabib** [comment]
can you invite me to the wechat group?

**Comment by enggabrielhabib** [comment]
can you invite me to the wechat group?
